io/ilvis2/Ilvis2Reader.cpp

```diff
@@ -123,11 +125,83 @@ T convert(const StringList& s, const std::string& name, size_t fieldno)
 }
 
 
+// If longitude between 0-180, just return it, degrees east; if between 180
+// and 360, subtract 360 to get negative value.
+double Ilvis2Reader::convertLongitude(double longitude)
+{
+  if (longitude > 180.0)
+  {
+    return longitude - 360.0;
+  }
+  else
+  {
+    return longitude;
+  }
+}
+
+
+void Ilvis2Reader::readPoint(PointViewPtr view, PointId nextId, StringList s, std::string pointMap)
+{
+  PointLayoutPtr layout = view->layout();
+
+  std::string name("LVIS_LFID");
```

Comment:
I could do that, although that would mean adding 6 new dimensions (lat/lon for high/low/centroid), and they do seem kind of LVIS-specific.  I may leave it for now, unless there are really strong feelings for making them general dimensions.